app/views/images/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Your image</h1>
 
-<%= image_tag(@image.link) %>
+<%= image_tag(@image.link, width: 400) %>
```

Comment:
consider using 'max-width' style here instead of width. Width can sometimes cause the image to stretch and get distorted, especially if we add height constraints later. Check out a couple explanations of the differences here, so you can at least come to understand the trade-offs: https://www.w3schools.com/Css/css_rwd_images.asp https://blog.prototypr.io/what-even-is-the-difference-between-width-and-max-width-8f37b282c7f1